### ISSUE TITLE
Specify the color space in all numpy.org CSS files

### ISSUE THREAD

**Issue by InessaPawson** [numpy/numpy.org#397]
Specify the color space (sRGB) in all numpy.org CSS files.

**Comment by sm-Fifteen** [comment]
The CSS specification says browsers should always use sRGB color space, but inconsistent support for this has been puzzling users for many years now (look up "chrome enable-color-correct-rendering" and "chrome force-color-profile" for a few examples). [I believe this is supposed to illustrate the issue on misbehaving browsers](http://www.libpng.org/pub/png/colorcube/colorcube-pngs-sRGB-CSS.html) (Firefox has some of those squares show up with a slightly duller middle on my screen, which would mean that it does not treat CSS colors as sRGB like it does with images, [see this bug on their tracker](https://bugzilla.mozilla.org/show_bug.cgi?id=999600)).

Explicitly color-managed CSS in the form of `color(sRGB 1.0 1.0 1.0)` isn't currently widely supported in browsers (I believe only Safari supports it at the moment) but [one can write CSS to fallback on default colors if the browser does not support it](https://css-tricks.com/the-expanding-gamut-of-color-on-the-web/#defining-fallbacks). To my knowledge, there is no other way of specifying color profiles in CSS.

**Comment by itscoolvatsa** [comment]
hey @InessaPawson  is this issue still open. I wanted to make my first contribution if it is?

**Comment by InessaPawson** [comment]
Hi, @itscoolvatsa! It looks like @aerikpawson is already working on it. Let's ping him to see if he is planning to finish the task.

**Comment by itscoolvatsa** [comment]
> Hi, @itscoolvatsa! It looks like @aerikpawson is already working on it. Let's ping him to see if he is planning to finish the task.

Yeah, sure btw I'm unfamiliar with the process, and I don't know how to do so.

**Comment by InessaPawson** [comment]
> Yeah, sure btw I'm unfamiliar with the process, and I don't know how to do so.

@itscoolvatsa Asking first is the best. So well done there!

**Comment by itscoolvatsa** [comment]
> > Yeah, sure btw I'm unfamiliar with the process, and I don't know how to do so.
> 
> @itscoolvatsa Asking first is the best. So well done there! 
> 
> 
One last thing when do I recheck fo this issue if available or not?

**Comment by aerikpawson** [comment]
@InessaPawson @itscoolvatsa I apologize for taking so long. I'll finish it within the next 2-3 days.

**Comment by InessaPawson** [comment]
Thank you for letting us know, @aerikpawson!

@itscoolvatsa We will have a low-code mini-sprint focusing on accessibility at the next Newcomer’s Hour scheduled for July 29th at 8PM UTC. This is a great opportunity to make your first contribution to NumPy. Here is more info about it: https://mail.python.org/pipermail/numpy-discussion/2021-July/081984.html. I hope you can make it.

**Comment by itscoolvatsa** [comment]
> Thank you for letting us know, @aerikpawson!
> 
> @itscoolvatsa We will have a low-code mini-sprint focusing on accessibility at the next Newcomer’s Hour scheduled for July 29th at 8PM UTC. This is a great opportunity to make your first contribution to NumPy. Here is more info about it: https://mail.python.org/pipermail/numpy-discussion/2021-July/081984.html. I hope you can make it.

Yeah sure I will

**Comment by InessaPawson** [comment]
> > @itscoolvatsa We will have a low-code mini-sprint focusing on accessibility at the next Newcomer’s Hour scheduled for July 29th at 8PM UTC. This is a great opportunity to make your first contribution to NumPy. Here is more info about it: https://mail.python.org/pipermail/numpy-discussion/2021-July/081984.html. I hope you can make it.
> 
> Yeah sure I will, can I get recordings of this meeting time, it's 1:30 am here😳

It will be @melissawm hosting it. I'm not sure we will be recording (usually we don't). Let’s wait for Melissa to respond.

**Comment by melissawm** [comment]
Hi @itscoolvatsa we won't be recording this but two weeks from now we might do another one at 4pm UTC, would you be able to make it then? (this is usually announce in the mailing list)

**Comment by itscoolvatsa** [comment]
> Hi @itscoolvatsa we won't be recording this but two weeks from now we might do another one at 4pm UTC, would you be able to make it then? (this is usually announce in the mailing list)

Yeah that one I can do, and where do I get notified for these events. And one last thing is it for contributing to the numpy library or numpy webpage.

**Comment by melissawm** [comment]
They are announced in the mailing list, and they are for contributing to the NumPy package (in this case, the documentation).

**Comment by itscoolvatsa** [comment]
> They are announced in the mailing list, and they are for contributing to the NumPy package (in this case, the documentation).

Thanks for the info.

**Comment by InessaPawson** [comment]
@aerikpawson LGTM (Looks good to me.) One suggestion: next time, before pushing, squash all the commits you'd like to include in a pull request into one. You’ll have to use the terminal for that.
